Cabal/Distribution/PackageDescription/Check.hs

```diff
@@ -1002,13 +1002,10 @@ checkCabalVersion pkg =
         ++ "at least 'cabal-version: >= 1.21'."
 
     -- check use of thinning and renaming
-  , checkVersion [1,21] (not (null depsUsingThinningRenamingSyntax)) $
+  , checkVersion [1,25] usesBackpackIncludes $
       PackageDistInexcusable $
-           "The package uses "
-        ++ "thinning and renaming in the 'build-depends' field: "
-        ++ commaSep (map display depsUsingThinningRenamingSyntax)
-        ++ ". To use this new syntax, the package needs to specify at least"
-        ++ "'cabal-version: >= 1.21'."
+           "To use the 'backpack-includes' field the package needs to specify "
+        ++ "at least 'cabal-version: >= 1.25'."
```

Comment:
Are we getting rid of thinning and renaming in the build-depends? Are there released packages using this or can we just drop it?